lib/ferver/controller.rb

```diff
@@ -36,14 +36,10 @@ class Controller < Sinatra::Base
     get '/files/:id' do
       halt(400, 'Bad request') unless valid_file_request?
 
-      if ferver_list.file_id_valid?(file_id_request.value)
-        file_name = ferver_list.file_by_id(file_id_request.value)
-        file = FileList.path_for_file(current_ferver_path, file_name)
-
-        send_file(file, disposition: 'attachment', filename: File.basename(file))
-      else
-        status 404
-      end
+      find_file
+      send_file(
+        @file.path_to_file, disposition: 'attachment', filename: @file.name
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.